src/kili/entrypoints/mutations/asset/__init__.py

```diff
@@ -140,14 +140,14 @@ def append_many_to_dataset(
         for key, value in field_mapping.items():
             if value is not None:
                 assets = [{**assets[i], key: value[i]} for i in range(nb_data)]
-        result = import_assets(
+        created_assets_ids = import_assets(
             self,
             project_id=project_id,
             assets=assets,
             disable_tqdm=disable_tqdm,
             verify=wait_until_availability,
         )
-        return result
+        return {"id": project_id, "created_assets_ids": created_assets_ids}
```

Comment:
I would just name the key `asset_ids`

Reply:
it was to make sure that the client understands that we don't return all the project asset ids

Reply:
But I am also good with just asset_ids

Reply:
ah ok makes sense, you can keep it as is then

Reply:
I used asset_ids, I also hesitated to put it as so